vm/src/thread.rs

```diff
@@ -560,7 +564,11 @@ impl ThreadInternal for Thread {
 
 
     /// Calls a module, allowed to to run IO expressions
-    fn call_module(&self, typ: &ArcType, closure: GcPtr<ClosureData>) -> Result<Value> {
+    fn call_module(&self,
+                   typ: &ArcType,
+                   closure: GcPtr<ClosureData>,
+                   run_io: bool)
+                   -> Result<Value> {
```

Comment:
You are right, I was a bit sloppy there since I figured it is just an internal function, not much more work to do it properly though. 😓